### PR TITLE
changed way bam files are imported because the regex was incorrect wh…

### PR DESCRIPTION
**Pull request by Acribbs**
…en ran for a second time

### CODE REVIEW

**Comment by Acribbs** [comment]
@sebastian-luna-valero I have made these changes to simplify the import of the data and fix various bugs

File: CGATPipelines/pipeline_bamstats/pipeline.ini [hunk 1]
```diff
@@ -63,7 +57,7 @@ name=csvdb
 #database location 
 database=/ifs/projects/adam/pipeline_publication/csvdb
```
Comment (sebastian-luna-valero) [inline CGATPipelines/pipeline_bamstats/pipeline.ini]: I think we should store this information in `/ifs/mirror` or similar but not on a project specific folder

File: CGATPipelines/pipeline_bamstats/pipeline.ini [hunk 2]
```diff
@@ -63,7 +57,7 @@ name=csvdb
...
 
-# location that pipeline_bamstats.py was ran
+# location that pipeline_genesets.py was ran in
 dir=/ifs/projects/adam/pipeline_publication
```
Comment (sebastian-luna-valero) [inline CGATPipelines/pipeline_bamstats/pipeline.ini]: I think we should store this information in `/ifs/mirror` or similar but not on a project specific folder

**Review by sebastian-luna-valero — COMMENTED** [review]
Thanks @Acribbs 

Will run jenkins tests to double check.

File: CGATPipelines/pipeline_bamstats/pipeline.ini (hunk 1 above)
Comment (Acribbs) [reply CGATPipelines/pipeline_bamstats/pipeline.ini]: I agree, will make changes and push again

File: CGATPipelines/pipeline_bamstats/pipeline.ini (hunk 2 above)
Comment (Acribbs) [reply CGATPipelines/pipeline_bamstats/pipeline.ini]: Ok agreed, ill push new changes

**Comment by sebastian-luna-valero** [comment]
Thanks @Acribbs 

Merging now, but will report back if the tests fail

**Pull request merged**